home/screens/ProjectsScreen.tsx

```diff
@@ -280,40 +286,43 @@ export default class ProjectsScreen extends React.Component {
     );
   };
 
-  _renderRecentHistoryItems = () => {
+  private _renderRecentHistoryItems = () => {
     const extractUsername = manifestUrl => {
-      let username = manifestUrl.match(/@.*?\//)[0];
+      const username = manifestUrl.match(/@.*?\//)[0];
       if (!username) {
         return null;
       } else {
         return username.slice(0, username.length - 1);
       }
     };
 
-    return this.props.recentHistory.map((project, i) => (
-      <SmallProjectCard
-        key={project.manifestUrl}
-        iconUrl={project.manifest && project.manifest.iconUrl}
-        releaseChannel={
-          /* 28/11/17(brentvatne) - we can remove extractReleaseChannel in a couple of months
+    return this.props.recentHistory.map(
+      (project, i) =>
+        project && (
+          <SmallProjectCard
+            key={project.manifestUrl}
+            iconUrl={project.manifest?.iconUrl}
+            releaseChannel={
+              /* 28/11/17(brentvatne) - we can remove extractReleaseChannel in a couple of months
           when project history is unlikely to include any projects with release channels */
-          (project.manifest && project.manifest.releaseChannel) ||
-          extractReleaseChannel(project.manifestUrl)
-        }
-        platform={project.platform}
-        projectName={project.manifest && project.manifest.name}
-        username={
-          project.manifestUrl.includes('exp://exp.host')
-            ? extractUsername(project.manifestUrl)
-            : null
-        }
-        projectUrl={project.manifestUrl}
-        fullWidthBorder={i === this.props.recentHistory.count() - 1}
-      />
-    ));
+              (project.manifest && project.manifest.releaseChannel) ||
+              extractReleaseChannel(project.manifestUrl)
+            }
+            platform={project.platform}
```

Comment:
@brentvatne History items don't seem to have a `platform` field. Do you know whether they once did? Shall I remove this?

![image](https://user-images.githubusercontent.com/6184593/79959385-5f70d100-8484-11ea-96f6-a2061212baf9.png)

Reply:
hm 🤔 the last time i touched this code was probably nearly 3 years ago - it does seem like it may not have existed before. let's remove it.

Reply:
I checked, and it wasn't doing anything anyway. Thanks for letting me know, I've removed it 👍